papermill/cli.py

```diff
@@ -10,10 +10,18 @@
 
 from .execute import execute_notebook
 from .iorw import read_yaml_file
+from .api import Notebook as notebook
 
 click.disable_unicode_literals_warning = True
 
 
+def print_the_paermill_version(ctx, param, value):
```

Comment:
`print_papermill_version`

Reply:
made the change

Reply:
I meant we should rename the function :)

Reply:
@MSeal can you please suggest any name for changing the name of function?

Reply:
@MSeal made the change and committed. Now it's looking cool.